### PR TITLE
Add missing creators in free-courses-fr.md

### PR DESCRIPTION
**Pull request by Mamadou442**
## What does this PR do?
Improve repo

### Description

- Add the website name (it includes the caps - [source](http://html.net/about/)) and the translator for fr.html.net links.
- Update "(Tutoriels Videos)" (it includes a capital letter and is format like a domain name - [source](https://www.tutoriels-video.fr/a-propos/)) and add the author for Tutoriels-video.fr links.
- Fix typo (Tutoriel HTML --> Tutoriel PHP).

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-fr.md [hunk 1]
```diff
@@ -116,7 +116,7 @@
 
 ### Linux
 
-* [Linux et Ubuntu - Administration Réseau](https://www.tutoriels-video.fr/category/ubuntu/) (Tutoriels Videos)
+* [Linux et Ubuntu - Administration Réseau](https://www.tutoriels-video.fr/category/ubuntu/) - Tutoriels-video.fr (Alexis Madrzejewski)
```
Comment (eshellman) [inline courses/free-courses-fr.md:119]:
```suggestion
* [Linux et Ubuntu - Administration Réseau](https://www.tutoriels-video.fr/category/ubuntu/) - Alexis Madrzejewski
```

File: courses/free-courses-fr.md [hunk 2]
```diff
@@ -151,4 +151,4 @@
 
 ### SysAdmin
 
-* [Développement Web - Administration Réseau](https://www.tutoriels-video.fr/category/webdev/) (Tutoriels Videos)
+* [Développement Web - Administration Réseau](https://www.tutoriels-video.fr/category/webdev/) - Tutoriels-video.fr (Alexis Madrzejewski)
```
Comment (eshellman) [inline courses/free-courses-fr.md:154]:
```suggestion
* [Développement Web - Administration Réseau](https://www.tutoriels-video.fr/category/webdev/) - Alexis Madrzejewski
```

File: courses/free-courses-fr.md [hunk 3]
```diff
@@ -79,8 +79,8 @@
 * [HTML/CSS - Exercices](https://www.youtube.com/playlist?list=PLrSOXFDHBtfHEFVqv0pjGkPHv6PhWZQBb) - Formation Video
 * [HTML/CSS - Tutoriels](https://www.youtube.com/playlist?list=PLrSOXFDHBtfG1_4HrfPttdwF8aLpgdsRL) - Formation Video
 * [Le préprocesseur SASS](https://www.youtube.com/playlist?list=PLjwdMgw5TTLWVp8WUGheSrGnmEWIMk9H6) - Grafikart
-* [Tutoriel CSS](http://fr.html.net/tutorials/css/)
-* [Tutoriel HTML](http://fr.html.net/tutorials/html/)
+* [Tutoriel CSS](http://fr.html.net/tutorials/css/) - HTML.net, `trl.:` Jean Jacques Solari
```
Comment (eshellman) [inline courses/free-courses-fr.md:82]:
```suggestion
* [Tutoriel CSS](http://fr.html.net/tutorials/css/) - `trl.:` Jean Jacques Solari
```

File: courses/free-courses-fr.md [hunk 4]
```diff
@@ -79,8 +79,8 @@
...
+* [Tutoriel HTML](http://fr.html.net/tutorials/html/) - HTML.net, `trl.:` Jean Jacques Solari
```
Comment (eshellman) [inline courses/free-courses-fr.md:83]:
```suggestion
* [Tutoriel HTML](http://fr.html.net/tutorials/html/) - `trl.:` Jean Jacques Solari
```

File: courses/free-courses-fr.md [hunk 5]
```diff
@@ -133,7 +133,7 @@
 * [Apprendre Symfony 4 par l'exemple](https://www.youtube.com/playlist?list=PLjwdMgw5TTLX7wmorGgfrqI9TcA8nMb29) - Grafikart
 * [Cours complet de Pierre Giraud sur le développement PHP avec MySQL](https://www.pierre-giraud.com/php-mysql-apprendre-coder-cours/) - Pierre Giraud
 * [Tester sur Symfony](https://www.youtube.com/playlist?list=PLjwdMgw5TTLWtWmdMzPaoc45Iztu7tVQ8) - Grafikart
-* [Tutoriel HTML](http://fr.html.net/tutorials/php/)
+* [Tutoriel PHP](http://fr.html.net/tutorials/php/) - HTML.net, `trl.:` Jean Jacques Solari
```
Comment (eshellman) [inline courses/free-courses-fr.md:136]:
```suggestion
* [Tutoriel PHP](http://fr.html.net/tutorials/php/) - `trl.:` Jean Jacques Solari
```

**Review by eshellman — CHANGES_REQUESTED** [review]
here the websites are not platforms, they're more like a publisher - we list the platform because there are platform affordances, we don't list publishers. Yes, this is an obscure distinction, and we don't apply the rule very uniformly, but I prefer to highlight the creators/translators.

**Comment by Mamadou442** [comment]
I see what you mean. I add all the changes into a single commit again. Thank for your review!

**Pull request merged**